packages/components/psammead-useful-links/src/index.stories.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs } from '@storybook/addon-knobs';
+import { inputProvider } from '@bbc/psammead-storybook-helpers';
+import {
+  UsefulLink,
+  UsefulLinksLi,
+  UsefulLinksUl,
+  UsefulLinkItem,
+} from './index';
+import SectionLabel from '../../psammead-section-label/src';
+import notes from '../README.md';
+
+const singleItem = ['Labaran BBC Hausa a text'];
+const multipleItems = [
+  'Mitocinmu da sauko da sautin labarai',
+  'Labaran BBC Hausa a text',
+  'Abokan huldar BBC Hausa',
+  'Timi Frank: Osinbajo ya maka mutum biyu',
+  'Gwaninta ba ta karbi wani dan Nijeriya',
+];
+
+const generateStory = ({ usefulItems, strapline }) =>
+  inputProvider({
+    // eslint-disable-next-line react/prop-types
+    componentFunction: ({ script, service }) => {
+      return (
+        <div>
+          {strapline ? (
```

Comment:
We could improve this by
```suggestion
          {strapline && (
``` 
And eliminate the need for `null`